Dockerfile_dev

```diff
@@ -6,11 +6,11 @@ ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSG
 ## System preparation steps ################################################# ##
 
 # !!! Runtime changes won't affect requirements.txt
-COPY requirements.txt /tmp/
+COPY requirements.txt requirements_test.txt /tmp/
 # * Install system requirements
 # * Install pip requirements
 # * Empty system cache to conserve some space
-RUN zypper -n in python310-devel gcc libffi-devel && pip3.10 install -r /tmp/requirements.txt && rm -rf /var/cache
+RUN zypper -n in python310-devel gcc libffi-devel && pip3.10 install -r /tmp/requirements_test.txt && rm -rf /var/cache
```

Comment:
I usually add `--no-cache-dir` to `pip`.

Reply:
ok , good to know but I assume it does not make much difference when there is `rm -rf /var/cache` , right ?